exalearn-ui/src/app/services/roles.service.ts

```diff
@@ -0,0 +1,47 @@
+import {Injectable} from '@angular/core';
+import {RedirectBtn} from "../interfaces/interfaces";
+
+@Injectable({
+  providedIn: 'root'
+})
+export class RolesService {
+  private userBtns: readonly RedirectBtn[] = [
+    {name: "my tests history", url: "/history"},
+    {name: "pass test", url: "/test"},
```

Comment:
We need to add pipe inside *ngFor to make buttons names uppercase, or at least rewrite names  with uppercase first letter: 'pass test' -> 'Pass test'

Reply:
added pipe